doc/api/repl.md

```diff
@@ -421,6 +421,9 @@ Returns `true` if `keyword` is a valid keyword, otherwise `false`.
 <!-- YAML
 added: v0.1.91
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
```

Comment:
This was landed with a mistake.
I will make a PR with a fix (with some other minor link fixes in the docs).